_pages/platform/usage-metrics.md

```diff
@@ -0,0 +1,69 @@
+- algorithm: algorithm display name
+- version: hash version of the exact algorithm called
+- caller: which user called the algorithm
+- owner: user who created and owns the algorithm
+- total calls: total number of calls the calling user made to this algorithm in the date range
+- total duration: total number of seconds the algorithm ran for caller in the date range
+- total errors: total number of failed calls to this algorithm the caller made in the date range
+- error duration: total number of seconds the algorithm ran for calls in which it returned an error (for this caller)
+- hardware: whether the algorithm runs on a CPU or GPU instance
+
+One caveat to the total compute time: each call is rounded **up** to the nearest second. For instance, if in the given date range, a user calls a certain algorithm twice, and the first call took 2.04 seconds and the second call took 2.08 seconds, the total time for those two calls is 4.12 seconds, which the table will round up and display as 5 seconds. 
+
+Also note that if a call takes less than one second, the call’s duration will be rounded up to a full second. So calling Hello World 23 times would most likely result in a total duration of 23 seconds being displayed. The motivation for this is that in the algorithm marketplace we only charge by credits. One credit = one second, so if a user makes a call that lasts less than a second we still charge for the full second.
+
+
+# Algorithms
+
+This tab shows usage information grouped at the algorithm level. That is, it shows which algorithms were called and how many times (total across all users) they were called.
+
+When you click on a row in the algorithms table, you will see this complete set of columns:
+- algorithm: algorithm display name
+- version: hash version of the exact algorithm called
+- owner: user who created and owns the algorithm
+- total calls: total number of calls that users of the platform made to this algorithm
+- total duration: total number of seconds the algorithm ran summed up across all users
+- total errors: total number of errors this algorithm returned summed up across all users
+- error duration: total number of seconds the algorithm ran calls in which it returned an error (summed up across all users)
+- hardware: whether the algorithm runs on a CPU or GPU instance
+
+
+# Users
+
+This tab shows usage information grouped at the user level, so you can see how many total calls each user made (across all algorithms).
+
+When you click on a row in the users table, you will see this complete set of columns:
+caller: the username of the user this row represents
+total calls: total number of calls the user made summed up across all algorithms
+total duration: total number of seconds the user’s calls spent computing summed up across all algorithms
+total errors: total number of errors returned to the user summed up across all algorithms
+error duration: total number of seconds computed by calls that returned errors to the user
```

Comment:
<img width="660" alt="Screen Shot 2020-01-29 at 4 06 17 PM" src="https://user-images.githubusercontent.com/8980868/73408509-4f948700-42b1-11ea-863b-34c69f3da55b.png">
This part's rendering a bit funky. Looks like it just needs the dashes.